ios/FluentUI/Avatar/AvatarTokenSet.swift

```diff
@@ -231,9 +239,49 @@ extension AvatarTokenSet {
             return GlobalTokens.iconSize(.small)
         }
     }
+
+    /// The border radius for the activity icon in the MSFAvatarActivityStyle `.square` style.
+    static func activityIconRadius(_ size: MSFAvatarSize) -> CGFloat {
+        switch size {
+        case .size16, .size20, .size24, .size32:
+            return 0
+        case .size40, .size56:
+            return GlobalTokens.borderRadius(.medium)
+        case .size72:
+            return GlobalTokens.borderRadius(.large)
+        }
+    }
+
+    /// The size of the activity icon.
+    static func activityIconSize(_ size: MSFAvatarSize) -> CGFloat {
+        switch size {
+        case .size16, .size20, .size24, .size32:
+            return 0
+        case .size40:
+            return GlobalTokens.iconSize(.xSmall)
+        case .size56:
+            return GlobalTokens.iconSize(.medium)
+        case .size72:
+            return 32
+        }
+    }
+
+    /// The size of the activity icon background.
+    static func activityIconBackgroundSize(_ size: MSFAvatarSize) -> CGFloat {
+        switch size {
+        case .size16, .size20, .size24, .size32:
+            return 0
```

Comment:
Does this mean we'll still be showing the activity, it's just of size 0?

Reply:
Yes, this is how we handle the presence as well since we have to make sure all the cases are addressed